pkg/consts/consts.go

```diff
@@ -1,7 +1,7 @@
 package consts
 
 const (
-	MaxOcpVersion = "4.14" // Latest supported version (update on each release)
+	MaxOcpVersion = "4.15" // Latest supported version (update on each release)
```

Comment:
why does max required? isn't minimal version enough?

Reply:
We maintain the max version for retesting the appliance on every minor version. As we might need to add additional images to oc-mirror for bootstrap, so it's not self maintained... But at least we get a clean fallback to the latest supported version by having a constant.